lib_forceload/data/gm4_forceload-1.0/functions/mark_dimension.mcfunction

```diff
@@ -4,12 +4,11 @@
 function gm4_forceload-1.0:init_chunk
 
 # spawn dimension marker
-summon minecraft:area_effect_cloud 29999999 0 7135 {Duration:2147483647,Tags:["gm4_dimension","gm4_new_dimension"]}
+summon minecraft:marker ~ 0 ~ {Tags:["gm4_dimension","gm4_new_dimension"]}
```

Comment:
This function runs at worlds spawn or the player's position from the advancement, why are the absolute coordinates removed?

Reply:
It needs to be spawned in a loaded position in order to perform `data modify` commands on it. It can't be found if it's spawned in a newly force loaded chunk.